core/src/main/java/com/amplifyframework/hub/HubEvent.java

```diff
@@ -125,6 +127,14 @@ public UUID getId() {
         return uuid;
     }
 
+    /**
+     * Publish the instance of the {@link HubEvent} to Amplify Hub.
+     * @param channel The channel to publish the event to.
+     */
+    public void publish(@NonNull HubChannel channel) {
+        Amplify.Hub.publish(channel, this);
```

Comment:
Eh, I feel iffy about binding `Amplifly`, here. I like the intention of this utility method though. Could you instead have it like:
```java
public void publish(@NonNull HubChannel toChannel, @NonNull HubCategoryBehavior usignHubInstance) {
    usingHubInstance.publish(channel, this);
}
```
?

Reply:
That does feel kind of out of place here. I'm going to move this.